src/System.Runtime.CompilerServices.VisualC/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap-Windows_NT;
```

Comment:
I don't see why src\configurations.props contains a uap configuration: if we can remove it, you won't need this. @weshaggard ?